stack/stack.go

```diff
@@ -20,6 +20,12 @@ import (
 
 const providerName = "faas"
 const providerNameLong = "openfaas"
+const defaultSchemaVersion = "1.0"
+
+// ValidSchemaVersions available schema versions
+var ValidSchemaVersions = []string{
```

Comment:
I imagined this is where all the supported versions would be listed and it would be updated when a new version is added. I would be happy to move it though. Any recommendations for a more appropriate place?